### PR TITLE
(fix): docs link

### LINKED ISSUE #2962: In "Getting Started" documentation broken link to https://www.fastify.io/docs/latest/Ecosystem
Hi.

On the page https://www.fastify.io/docs/latest/Getting-Started/#extend-your-server the link to the page https://www.fastify.io/docs/latest/Ecosystem is not working.
It is found in this sentence:

> In other words, Fastify is not a "batteries included" framework, and relies on an amazing ecosystem!

Greetings
Bernhard

### PR DESCRIPTION
**Pull request by makuzaverite**
fix  #2962 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ Read [Validation and Serialization](Validation-and-Serialization.md) to learn mo
 <a name="extend-server"></a>
 ### Extend your server
 Fastify is built to be extremely extensible and minimal, we believe that a bare bones framework is all that is necessary to make great applications possible.<br>
-In other words, Fastify is not a "batteries included" framework, and relies on an amazing [ecosystem](Ecosystem.md)!
+In other words, Fastify is not a "batteries included" framework, and relies on an amazing [ecosystem](https://www.fastify.io/ecosystem/)!
```
Comment (Eomm) [inline docs/Getting-Started.md:294]: I think a better fix would be to fix the website builder here

https://github.com/fastify/website/blob/06409ed51c972bc4d3682e00419f0807487b4cb5/src/scripts/processReleases.js#L144-L149

So the user that navitages in the site, will remains in the site and if the dev navigates through github will stay here

Comment (makuzaverite) [reply docs/Getting-Started.md:294]: Yeah this totally make sense

**Comment by makuzaverite** [comment]
Fixed @Eomm

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by Fdawgs** [comment]
Has this PR been superseded by another?
It only trims the trailing white space from an inline comment now. 😄

**Pull request closed without merging**